### ISSUE TITLE
preflight response does not include PATCH in the "Access-Control-Allow-Methods header"

### ISSUE THREAD

**Issue by mohitparmar1** [idurar/idurar-erp-crm#1193]
**Describe the bug**
The PATCH method triggers a CORS error when attempting to update the profile. The preflight response does not include PATCH in the Access-Control-Allow-Methods header.

**To Reproduce**
1. Go to https://cloud.idurarapp.com and log in.
2. Trigger a profile update that uses a PATCH request.
3. Check the console/network tab.

**Expected behavior**
The PATCH request should be allowed, and the update should proceed without a CORS error.

**Screenshots**
![Screenshot 2024-11-09 151352](https://github.com/user-attachments/assets/07a53fbf-3361-4ce8-bb5a-e841c48b695e)

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by mohitparmar1** [comment]
@salahlalami if it is a valid issue I would like to work on this!

**Comment by surjeet88** [comment]
Thanks  @mohitparmar1 for using the perfect format for describing the issue.I would love to work on it/

**Comment by mohitparmar1** [comment]
> Hi @mohitparmar1 i have a cloned copy of the code on my local machine there i have an admin user made through setup file when i update the profile in there then i am not seeing any error but in the deployed code there is indeed an error.It's my first time working on an open source project.Could you guide why this is happening?

Yes, but the changes are not updating there, suppose someone changed their last it is not showing new changes after the update

**Comment by surjeet88** [comment]
Here is my name before change
![Screenshot from 2024-11-10 12-01-28](https://github.com/user-attachments/assets/9e812558-fb4a-4c63-8fa2-6675ba56f90b)
and this is after
![Screenshot from 2024-11-10 12-01-42](https://github.com/user-attachments/assets/408c4922-11e2-4e1c-ae3b-e46992780a98)
@mohitparmar1

**Comment by surjeet88** [comment]
> > Hi @mohitparmar1 i have a cloned copy of the code on my local machine there i have an admin user made through setup file when i update the profile in there then i am not seeing any error but in the deployed code there is indeed an error.It's my first time working on an open source project.Could you guide why this is happening?
> 
> Yes, but the changes are not updating there, suppose someone changed their last name it is not showing new changes after the update

I saw your PR you added the method parameter to the cors middleware it is necessary for patch.I am new to open source as i could understand you changed just one line is that correct or you made some other changes?
